pyuploadtool/changelog/parsers/markdown.py

```diff
@@ -0,0 +1,32 @@
+from .parser import ChangelogParser
+
+
+class MarkdownChangelogParser(ChangelogParser):
+    def render_to_markdown(self):
+        """
+        Get Markdown MarkdownChangelogParser
```

Comment:
Pretty broken docstring. The fields below are not even filled in.

Reply:
Fixed in https://github.com/TheAssassin/pyuploadtool/pull/3/commits/db0952b6011c414d4a4b8e671cd3c8412aad04cf